test/scripts

```diff
@@ -0,0 +1 @@
+../scripts
```

Comment:
What is the purpose of this file?

Reply:
I don't like much the solution I came up with, but this is to allow importing the `convert_config.py` as a module and use inside the test to generate the configuration files.

Reply:
I think the best here would be to follow the [suggestion](https://github.com/keylime/keylime/pull/1133#issuecomment-1282730523) from @THS-on and install the `convert_config.py` script along the other utilities which will also make it a keylime submodule that can be imported without this _hackish_ solution.